src/frontend/src/lib/api/auth-client.api.ts

```diff
@@ -0,0 +1,3 @@
+import { IdbStorage } from '@dfinity/auth-client';
+
+export const idbStorage = new IdbStorage();
```

Comment:
- Should we rename it to express a bit better it's purpose, like `authClientStorage` or something of that sort?

- Should we add a short comment on why we delcare a storage? Even just `@see safeCreateAuthClient`?